### PR TITLE
Feature/scope change improvements

### PR DESCRIPTION
**Pull request by Gustav-Eikaas**
# Description

Please include a summary of the change and which issue is fixed.
List any external dependencies that are required for this change.

Completes: AB#FILL_IN_YOUR_ISSUE_ID

## Checklist:

-   [ ] I have performed a self-review of my own code.
-   [ ] I have linked my DevOps task using the AB# tag.
-   [ ] My code is easy to read, and comments are added where needed.
-   [ ] My code is covered by tests.

### CODE REVIEW

File: src/apps/ScopeChangeRequest/Api/STID/Types/Document.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+export interface Document {
+    // instCode: string;
+    docNo: string;
+    docTitle: string;
+    // docClass: string;
+    // projectCode: string;
+    // poNo?: string;
+    // system: string;
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Api/STID/Types/Document.ts:8]: remove comments

File: src/apps/ScopeChangeRequest/Api/STID/getDocumentById.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ * @param instCode
+ * @param docNo
+ * @param stidClient
+ * @returns
+ */
+export async function getDocumentById(
+    instCode: string,
+    docNo: string,
+    stidClient: BaseClient
+    //TODO:
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Api/STID/getDocumentById.ts:15]: wrong place

File: src/apps/ScopeChangeRequest/Components/DetailView/Components/Field.tsx [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import styled from 'styled-components';
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Components/DetailView/Components/Field.tsx:1]: remove

File: src/apps/ScopeChangeRequest/Components/DetailView/RequestDetailView.tsx [hunk 4]
```diff
@@ -149,20 +144,28 @@ export const RequestDetailView = ({
...
+                    label={'System / comm pkg / tag'}
+                    value={<div></div>}
+                />
+                <Field
+                    customLabel={{ fontSize: '18px', bold: true }}
+                    label={'Documents'}
+                    value={<StidDocumentResolver inputDocuments={request.documents} />}
+                />
+                <Field
+                    customLabel={{ fontSize: '18px', bold: true }}
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Components/DetailView/RequestDetailView.tsx:159]: Delete

File: src/apps/ScopeChangeRequest/Components/SearchableDropdown/stidSelector.tsx [hunk 5]
```diff
@@ -18,19 +20,45 @@ interface StidSelectorProps {
 export const StidSelector = ({ appendDocuments }: StidSelectorProps): JSX.Element => {
     const [isOpen, setIsOpen] = useState(false);
     const [documents, setDocuments] = useState<TypedSelectOption[]>([]);
+    const [tagContainsNoDocuments, setTagContainsNoDocuments] = useState<boolean>(false);
+    //prod
     const { customApi } = useApiClient('1734406c-3449-4192-a50d-7c3a63d3f57d/.default');
+    //test
+    // const { customApi } = useApiClient('3aa4a235-b6e2-48d5-9195-7fcf05b459b0');
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Components/SearchableDropdown/stidSelector.tsx:27]: Yep

File: src/apps/ScopeChangeRequest/Components/SearchableDropdown/stidSelector.tsx [hunk 6]
```diff
@@ -80,31 +108,58 @@ export const StidSelector = ({ appendDocuments }: StidSelectorProps): JSX.Elemen
...
+                                        addDocument(actionMeta.option);
+                                    } else {
+                                        const documents = await getDocumentsByTag(
+                                            'JCA',
+                                            actionMeta.option.value,
+                                            customApi
+                                        );
+                                        if (documents.length <= 0) setTagContainsNoDocuments(true);
+                                        documents.forEach((x) => {
+                                            if (x && x.docNo) {
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Components/SearchableDropdown/stidSelector.tsx:126]: extract to function

File: src/apps/ScopeChangeRequest/Components/StidDocument.tsx [hunk 7]
```diff
@@ -14,8 +14,10 @@ export const StidDocument = ({ document }: StidDocumentProps): JSX.Element => {
         window.open(`https://lci.equinor.com/JCA/doc?docNo=${docNo}`);
     };
     return (
-        <Inline>
-            <Icon name="file_copy" color={tokens.colors.interactive.primary__resting.rgba} />
+        <Wrapper>
+            <div style={{ width: '24px', height: '24px' }}>
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Components/StidDocument.tsx:18]: Styled components

File: src/apps/ScopeChangeRequest/Components/Workflow/WorkflowLine.tsx [hunk 8]
```diff
@@ -24,6 +24,7 @@ export const WorkflowLine = ({ colored }: WorklflowLineProps): JSX.Element => {
 
 const VerticalLine = styled.div`
     width: 24px;
-    border: 1px solid;
+    border: 1px solid grey;
     transform: rotate(90deg);
+    background-color: grey;
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Components/Workflow/WorkflowLine.tsx:29]: investigate

**Review by jaysencpp — APPROVED** [review]

**Pull request merged**